internal/bus/bus.go

```diff
@@ -1,17 +1,33 @@
+/*
+Package bus provides access to a singleton instance of an event bus (provided by the calling application). The event bus
+is intended to allow for the syft library to publish events for which library consumers can subscribe to. These events
```

Comment:
(nitpick on grammar)

>events for which library consumers can subscribe to.

to either of these:

_events to which library consumers can subscribe._

_events which library consumers can subscribe to._